Mathlib/CategoryTheory/Limits/Constructions/FiniteProductsOfBinaryProducts.lean

```diff
@@ -156,12 +157,17 @@ lemma preservesShape_fin_of_preserves_binary_and_terminal (n : ℕ) :
     apply preservesLimit_of_iso_diagram F that
 
 /-- If `F` preserves the terminal object and binary products then it preserves finite products. -/
-lemma preservesFiniteProducts_of_preserves_binary_and_terminal (J : Type*) [Finite J] :
+lemma preservesFiniteProducts.of_preserves_binary_and_terminal (J : Type*) [Finite J] :
```

Comment:
```suggestion
lemma PreservesFiniteProducts.of_preserves_binary_and_terminal (J : Type*) [Finite J] :
```
Or this should be called `preservesLimitsOfShape_discrete_of_preserves_binary_and_terminal` because this does not mention `PreservesFiniteProducts`.